### DISCUSSION TITLE
Syncing of Likes?

### DISCUSSION THREAD

**Discussion by hartman8227** [lyarenei/jellyfin-plugin-listenbrainz#65] [Q&A] [answered]
Does this plugin sync likes (hearts) from Jellyfin to ListenBrainz? I was playing around with this and noticed that the likes I make weren't being synced.

**Comment by lyarenei** [comment, marked as answer]
Yes, the plugin supports favorite syncing. However, it takes place only after a successful listen submission, not when you favorite/unfavorite the track in Jellyfin.

The system is described in more detail [here](https://github.com/lyarenei/jellyfin-plugin-listenbrainz/blob/main/doc/how-it-works.md#syncing-favorites).

Edit: since version 3.4.0.0 it is possible to sync favorites both ways (with some caveats). Also the favorites are synced immediately from Jellyfin (as long as there is a Track MBID available).
